transforms/angle-brackets/index.js

```diff
@@ -26,9 +28,40 @@ function getOptions() {
   return options;
 }
 
+function helperNames(helperPaths) {
+  return [
+    ...new Set(
+      helperPaths
+        .filter(name => name.includes('/helpers/') || name.includes('/helper'))
+        .filter(name => !name.includes('/-'))
+        .map(name => {
+          let path = name.split('/helpers/');
+          return path.pop();
+        })
+        .filter(name => !name.includes('/'))
+    ),
+  ];
+}
+
+function getHelperData(telemetry) {
+  let helperPaths = [];
+  let telemetryKeys = Object.keys(telemetry);
+  for (let path of telemetryKeys) {
+    let entry = telemetry[path];
+    if (entry.type === 'Helper') {
+      helperPaths.push(path);
+    }
+  }
+  let helpers = helperNames(helperPaths);
+  debug(JSON.stringify(telemetryKeys, null, 2));
```

Comment:
Probably best to prefix this debug statement with some indication of what it actually is printing 

e.g. 

```
debug(`known modules:\n${JSON.stringify(telemetryKeys, null, 2)}`);
```